iroh-p2p/src/node.rs

```diff
@@ -578,6 +595,26 @@ impl<KeyStorage: Storage> Node<KeyStorage> {
                         .ok();
                 }
             }
+            RpcMessage::StartProviding(response_channel, key) => {
+                if let Some(kad) = self.swarm.behaviour_mut().kad.as_mut() {
+                    let res: Result<QueryId> = kad.start_providing(key).map_err(|e| e.into());
```

Comment:
While this triggers the providing, nothing is actually provided when this returns.  Can you at least add a todo, to wait for the result of this query from kademlia?

Reply:
added a note

But also, did a quick version of this (waiting on kad to actually start providing) and it took around 30s for each block, which made `iroh-ctl add` agonizing to wait for. We should probably be doing the work to add the block to the store and providing the block to the network in a `task::spawn`, but didn't want to shove that into this pr. lmk if you have thoughts about this.